docs/docs/reference/dropped-features/macros.md

```diff
@@ -1,13 +1,12 @@
 ---
 layout: doc-page
-title: Dropped: Macros
+title: Dropped: Scala 2 Macros
 ---
 
 The previous, experimental macro system has been dropped. Instead,
 there is a cleaner, more restricted system based on two complementary
```

Comment:
Don't talk about "two complementary concepts" when the list below has three entries :).

Reply:
Modified text. Added more detail on how the parts interact without going in depth.